model/trace/database.yaml

```diff
@@ -251,3 +251,24 @@ groups:
         requirement_level:
           conditionally_required: when available
         tag: tech-specific
+
+  - id: db.influxdb
+    type: span
+    extends: trace.db.common
+    brief: >
+      Attributes for InfluxDB
+    attributes:
+      - ref: db.name
```

Comment:
Note: `db.name` is being changed in another PR: https://github.com/open-telemetry/semantic-conventions/pull/911